### PR TITLE
Added clients

### PR DESCRIPTION
**Pull request by gecofernando**
Added Google's and Fellow's logo.
Added Tango to the projects.

### CODE REVIEW

**Comment by adamantivm** [comment]
Did we already get authorization from Fellow @tulku ?

File: source/projects/google.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: Google's Tango
```
Comment (adamantivm) [inline source/projects/google.html.md.erb]: I'd suggest renaming this to something like: "Augmented Reality Applications with Google Tango"
@JessicaJackson wdyt?

Comment (gecofernando) [inline source/projects/google.html.md.erb]: Changed it. Sound good to me.

**Comment by adamantivm** [comment]
I'd prefer to have @tulku 's confirmation of approval for the Fellow case before publishing. Written would be ideal.

File: source/projects/google.html.md.erb (hunk 1 above)
Comment (JessicaJackson) [inline source/projects/google.html.md.erb]: Agreed!


Thanks!

Jessica

--
Jessica Jackson
*Project Manager*

AR: +54 (11) 5219-0895
US: +1 (408) 757-0064




On Thu, Apr 14, 2016 at 3:10 PM, Fernando Kopelman <notifications@github.com

**Comment by gecofernando** [comment]
I was saying I changed the title from the Google post.
@tulku was the one telling me to upload the FR logo. I'd wait a
re-confirmation anyways.

**Comment by tulku** [comment]
Yes, we did

**Comment by adamantivm** [comment]
deployed

**Pull request merged**